### PR TITLE
update chart version for ingress-nginx

### PR DESCRIPTION
**Pull request by MrBatschner**
**How to categorize this PR?**

/area quality
/kind enhancement
/priority 3

**What this PR does / why we need it**:

The ingress-nginx Helm chart that is included in the tutorials and used for (among others) for integration tests will be upgraded to to the latest version 3.29.0. The component-descriptors now include the Helm chart version instead of just _v0.1.0_.

**Special notes for your reviewer**:

**Release note**:

```other developer
- Helm chart for ingress-nginx upgraded to 3.29.0.
- hack/update-tutorial-resources.sh script now includes all new tutorial resources
```

### CODE REVIEW

File: hack/upload-tutorial-resources.sh [hunk 1]
```diff
@@ -20,23 +20,48 @@ then
...
-  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/simple-aggregated:${BLUEPRINT_AGGREGATED_VERSION},./docs/tutorials/resources/aggregated/blueprint"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/ingress-nginx;./docs/tutorials/resources/ingress-nginx/blueprint;v0.2.1"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/echo-server;./docs/tutorials/resources/echo-server/blueprint;v0.1.1"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/simple-aggregated;./docs/tutorials/resources/aggregated/blueprint;v0.1.1"
 )
 component_descriptors=(
   "./docs/tutorials/resources/ingress-nginx"
   "./docs/tutorials/resources/echo-server"
   "./docs/tutorials/resources/aggregated"
+  "./docs/tutorials/resources/local-ingress-nginx"
```
Comment (schrodit) [inline hack/upload-tutorial-resources.sh:37]: I think the external jsonschema tutorial is missing?

File: hack/upload-tutorial-resources.sh [hunk 2]
```diff
@@ -20,23 +20,48 @@ then
     exit
 fi
 
+# this array consists of <oci-path of blueprint>;<local path of blueprint>;<default version tag>
```
Comment (schrodit) [inline hack/upload-tutorial-resources.sh:23]: can you add a short section/newdoc in `docs/development` that explain how to add/update the resources and use that file

**Review by schrodit — CHANGES_REQUESTED** [review]
2 minor nits rest is lgtm

File: hack/upload-tutorial-resources.sh (hunk 2 above)
Comment (MrBatschner) [reply hack/upload-tutorial-resources.sh:19]: Done.

File: hack/upload-tutorial-resources.sh (hunk 1 above)
Comment (MrBatschner) [reply hack/upload-tutorial-resources.sh:33]: *grml* Forgot that one... fixed.

**Review by schrodit — CHANGES_REQUESTED** [review]
one last thing: can you add a link to the tutorial in the `docs/README.md`

**Comment by MrBatschner** [comment]
> one last thing: can you add a link to the tutorial in the `docs/README.md`

Of course, done.

**Review by schrodit — APPROVED** [review]
/lgtm

**Pull request merged**